client/my-sites/invites/invite-accept-logged-out/index.jsx

```diff
@@ -60,30 +60,61 @@ export default React.createClass( {
 		)
 	},
 
-	footerLink() {
-		let logInUrl = config( 'login_url' ) + '?redirect_to=' + encodeURIComponent( window.location.href );
+	subscribeUserByEmailOnly() {
+		this.setState( { submitting: true } );
+		acceptInvite(
```

Comment:
I think we should change what `acceptInvite` receives to a single object with multiple keys, what do you think?